src/bridge/components/StatelessTransaction.js

```diff
@@ -238,7 +239,7 @@ class StatelessTransaction extends React.Component {
         props.setTxnConfirmations
       )
         .then(txHash => {
-          props.onSent(Just(Ok(txHash)), state.stx.value);
```

Comment:
the second argument here was just ignored by the `onSent` function, so I'm removing it here.

Reply:
The reason the transaction is included here is that some flow implementations may depend on data from it. We previously had one such case, but I think that was refactored out?

I guess since it's not actively in use we should just remove it. If anything ever needs this, it can be added back in. (Though, better to be consistent about how transaction submission flows work.)